docs/concepts/dataclasses.md

```diff
@@ -265,6 +265,8 @@ class File:
 
 
 class Foo(BaseModel):
+    model_config = ConfigDict(revalidate_instances='always')
+    # Required so that pydantic revalidates the model attributes
```

Comment:
```suggestion
    # Required so that pydantic revalidates the model attributes
    model_config = ConfigDict(revalidate_instances='always')
    
```